PowerShell/JumpCloud Module/Public/Utilities/CSV_Import/Update-JCDeviceFromCSV.ps1

```diff
@@ -0,0 +1,110 @@
+Function Update-JCDeviceFromCSV () {
```

Comment:
Just an initial thought, this works just fine and I think like intended with the exception of this: when you go ahead and create a template with all the headers and submit update for just the device description, the function will fail.
![Screenshot 2024-12-17 at 12 17 51 PM](https://github.com/user-attachments/assets/9f814336-223d-4c45-a2ca-b2d40fb48147)

Would it be possible to check if there's a null value in the CSV to just ignore the item in each row?

Reply:
Got the same issue here as well. Ignoring a blank field/row would be ideal